ui/src/components/grader/DownloadResult.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import axios from 'axios'
+import fileDownload from 'js-file-download'
+
+
+function DownloadResult(props){
+
+    function downloadFile(){
+        axios.get(`http://localhost:5000/results/cc451/${props.lab}`).then(res => {
```

Comment:
please extract the website URL to a `config` file or something. Where it tries to fetch the variable from an environment variable first, then falls back to a default value